src/main.rs

```diff
@@ -95,7 +96,9 @@ fn parse_config<R: BufRead>(config: R) -> Vec<(String, Vec<String>)> {
     config
         .lines()
         .map(|x| x.unwrap())
+        // Filter out comments and blank lines
         .filter(|x| x.len() > 0)
+        .filter(|x| !x.starts_with("#"))
```

Comment:
Probably need to check without leading whitespace (`x.trim().starts_with('#')`), and check against a single character

Reply:
Fixed.